### PR TITLE
ref(event-errors): Use Alert instead of BannerSummary

### PR DESCRIPTION
**Pull request by vuluongj20**
Use the `<Alert />` component for event errors, instead of `BannerSummary` which mimics the Alert style.

**Before:**
Screen Shot 2022-04-04 at 10 44 17 AM
Screen Shot 2022-04-04 at 10 44 23 AM

**After:**
Screen Shot 2022-04-04 at 10 43 21 AM
Screen Shot 2022-04-04 at 10 43 28 AM

### CODE REVIEW

File: static/app/components/alert.tsx [hunk 1]
```diff
@@ -73,6 +73,11 @@ const alertStyles = ({
       text-decoration-style: solid;
     }
 
+    pre {
+      background: ${alertColors.backgroundLight};
+      margin: ${space(0.5)} 0 0;
+    }
+
```
Comment (vuluongj20) [inline static/app/components/alert.tsx:80]: Screen Shot 2022-04-04 at 11 31 28 AM

File: static/app/components/events/errors.tsx [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
 import {JavascriptProcessingErrors} from 'sentry/constants/eventErrors';
-import {IconWarning} from 'sentry/icons';
 import {t, tn} from 'sentry/locale';
-import space from 'sentry/styles/space';
 import {Artifact, Organization, Project} from 'sentry/types';
 import {Event} from 'sentry/types/event';
-import {Theme} from 'sentry/utils/theme';
 import withApi from 'sentry/utils/withApi';
 
-import {BannerContainer, BannerSummary} from './styles';
```
Comment (evanpurkhiser) [inline static/app/components/events/errors.tsx:21]: Can we delete these?

Comment (evanpurkhiser) [reply static/app/components/events/errors.tsx:21]: Oh maybe we did 👍

File: static/app/components/events/errors.tsx [hunk 3]
```diff
@@ -33,23 +29,17 @@ type Props = {
...
+  state: State = {};
 
   componentDidMount() {
     this.checkSourceCodeErrors();
   }
 
-  shouldComponentUpdate(nextProps: Props, nextState: State) {
-    if (this.state.isOpen !== nextState.isOpen) {
-      return true;
-    }
```
Comment (evanpurkhiser) [inline static/app/components/events/errors.tsx:52]: This was gross. Great cleanup

**Review by evanpurkhiser — COMMENTED** [review]
Code looks good. Will leave it to @robinrendle for that sweet sweet approval.

File: static/app/components/alert.tsx (hunk 1 above)
Comment (robinrendle) [reply static/app/components/alert.tsx:80]: Oh I like this lil detail 🌮

**Review by robinrendle — APPROVED** [review]
🌮 🌮 🌮 🌮

**Pull request merged**